openedx/features/journals/templates/journals/journal_card.html

```diff
@@ -0,0 +1,45 @@
+<%def name="online_help_token()"><% return "course" %></%def>
+<%namespace name='static' file='static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from django.core.urlresolvers import reverse
+from six import text_type
+from urlparse import urljoin
+%>
+<%page args="journal, journals_root_url" expression_filter="h"/>
+<article class="journal" id="${journal.get('slug')}" role="region" aria-label="${journal.get('title')}">
+  <a class="journals-listing-item" href="${urljoin(journals_root_url, journal.get('slug'))}">
+    <header class="journal-image">
+      <div class="cover-image">
+        <img src="${journal.get('card_image_url')}" alt="${journal.get('title')}" />
+        <div class="learn-more" aria-hidden="true">${_("LEARN MORE")}</div>
+      </div>
+    </header>
+    <div class="banner">
+      ${_("Journal")}
+    </div>
+
+    <div class="journal-info" aria-hidden="true">
+        <span class="journal-org">${journal.get('organization')}</span>
+        <h3 class="journal-title">${journal.get('title')}</h3>
+    </div>
+
+    <span class="journal-footer">
+        <%
+        if journal.get('access_length') is not None:
+          access_length_string = '{} months'.format(int(journal.get('access_length')/30))
```

Comment:
need to use `ungettext` here to both i18n and get the pluralization correct

Reply:
fixed